core/tests/test_apps.cardano.address_credential_policy.py

```diff
@@ -0,0 +1,255 @@
+from common import *
+
+from apps.cardano.helpers.address_credential_policy import (
+    CredentialPolicy,
+    get_payment_credential_policy,
+    get_stake_credential_policy,
+)
+from apps.common.paths import HARDENED
+from trezor.enums import CardanoAddressType
+from trezor.messages import CardanoAddressParametersType, CardanoBlockchainPointerType
+
+
+def _harden(path: list[int]) -> list[int]:
```

Comment:
I'd rather not introduce another method of "hardening". I'd prefer to use what's used in `test_apps.cardano.certificate.py`. This way it's not really clear what's being hardened on first sight - although yes, it's fairly obvious if you know what should be being hardened. But let's not assume everyone knows.

Reply:
Ok, fixed. I wanted to make it a bit more readable, but it doesn't really matter.